tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -31,6 +32,16 @@ function SplashPage() {
     <>
       <section className="usa-hero" aria-label="Introduction">
         <div className="grid-container">
+          {isInactive && (
+            <div className="usa-alert usa-alert--slim usa-alert--error">
+              <div className="usa-alert__body" role="alert">
+                <h3 className="usa-alert__heading">Inactive Account</h3>
+                <p className="usa-alert__text">
+                  Please email tanfdata@acf.hhs.gov to reactivate your account.
```

Comment:
TODO: Make this a link? Should it be prefixed with `mailto:`?

Reply:
Was this e-mail address provided by the client? I'm sure QASPreview would catch if it was incorrect but just wondering where this came from.
I do like your mailto link idea!

Reply:
It was provided in the design mockups linked in issue #829  https://www.figma.com/file/irgQPLTrajxCXNiYBTEnMV/TDP-Mockups-For-Feedback?node-id=4598%3A23260